### ISSUE TITLE
How to use the Makefile?

### ISSUE THREAD

**Issue by RidaAyed** [kaushalmodi/ox-hugo#359]
I need to setup test automation and therefore am looking at the Makefile. What am I missing in `make test/site/content-org/all-posts.org`? 

    cd /tmp
    git clone git@github.com:kaushalmodi/ox-hugo.git
    cd ox-hugo
    echo foo >> test/site/content-org/all-posts.org
    make test/site/content-org/all-posts.org

Output

    make: Nothing to be done for 'test/site/content-org/all-posts.org'.

**Comment by kaushalmodi** [comment]
If you type `make help` you will see all the options. 

Also see https://ox-hugo.scripter.co/doc/contributing-guide/ .

**Comment by kaushalmodi** [comment]
- `make md` will only export the tests from the `all-posts.org` to markdown files.
- `make -j1 test` will do above + check them again the **checked in** reference markdown files in `test/site/content/.

**Comment by RidaAyed** [comment]
tldr: How to to export from commandline?

Thank you. How can I test this function (from `make help`):

    make FOO.org       <-- Export the FOO.org file from content-org/ dir to Markdown file(s)
    make screenshot-subtree-export-example.org

Output

    make: *** No rule to make target 'FOO.org'.  Stop.
    make: *** No rule to make target 'screenshot-subtree-export-example.org'.  Stop.

**Comment by kaushalmodi** [comment]
You need to use a different internal make target `md1` with the file path **relative** to the ox-hugo repo.

Example: 

````
make md1 ORG_FILE=test/site/content-org/screenshot-subtree-export-example.org
```
